apiserver/plane/app/views/issue/custom_property.py

```
@@ -33,15 +34,62 @@
+                else:
+                    value_str = str(value).strip().lower()
+                    bool_value = value_str in ["true", "1", "yes"]
+                custom_property.bool_value = bool_value
+            case "date":
+                try:
+                    # Try to parse date in standard ISO format
+                    date_value = datetime.strptime(value, "%Y-%m-%d").date()
+                    custom_property.date_value = date_value
+                except (ValueError, TypeError):
+                    return custom_property, Response(
+                        {"error": f"Invalid date format. Date must be in YYYY-MM-DD format (e.g., 2025-04-24)."},
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)